tasty-query/shared/src/main/scala/tastyquery/Types.scala

```diff
@@ -642,8 +645,8 @@ object Types {
           case declaring: DeclaringSymbol =>
             val candidate = declaring.getDecl(name)
             candidate.getOrElse {
-              val msg = this match
-                case TermRef(_, name: SignedName) if declaring.memberPossiblyOverloaded(name) =>
+              val msg = this.name match
```

Comment:
its interesting to note here that the `name` when designator was a `LookupIn` will always be a `SignedName`, because it will be cached on this access here, before designator is replaced by a symbol